app/components/MigrationDataOverview.tsx

```diff
@@ -2,14 +2,18 @@ import type { Context } from "~/flows/contexts";
 import { type Translations } from "~/services/cms/index.server";
 import { getTranslationByKey } from "~/services/translations/getTranslationByKey";
 import { lookupOrKey } from "~/util/lookupOrKey";
+import Button from "./Button";
 import Heading from "./Heading";
 
 type MigrationDataProps = {
-  readonly migrationData?: Context;
+  readonly data?: Context;
   readonly translations: Translations;
-  readonly migrationOrderFields?: string[];
+  readonly orderFields?: string[];
```

Comment:
```suggestion
  readonly sortedFields?: string[];
```

fields that are sorted?